ZooRoyal/coding-standard
# Eslint filter fix

To Test this:
1) composer install this repo
2) Add a malformed js somewhere
3) src/bin/coding-standard sca:eslint should throw warnings
